src/meta/processors/schemaMan/AlterEdgeProcessor.cpp

```diff
@@ -56,7 +75,7 @@ void AlterEdgeProcessor::process(const cpp2::AlterEdgeReq& req) {
 
     // Update schema property
     auto& alterSchemaProp = req.get_schema_prop();
-    auto retCode = MetaServiceUtils::alterSchemaProp(columns, prop, std::move(alterSchemaProp));
```

Comment:
why remove `move`?

Reply:
req.get_schema_prop() return a const object , so there is no need for std::move